flink-formats/flink-sql-avro-confluent-registry/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,25 @@
+flink-sql-avro-confluent-registry
+Copyright 2014-2020 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- org.apache.avro:avro:1.10.0
+- com.fasterxml.jackson.core:jackson-core:2.10.1
+- com.fasterxml.jackson.core:jackson-databind:2.10.1
+- com.fasterxml.jackson.core:jackson-annotations:2.10.1
+- com.apache.commons:commons-compress:1.20
+- io.confluent:kafka-schema-registry-client:5.5.2
+- org.apache.kafka:kafka-clients:5.5.2-ccs
+- io.confluent:common-config:5.5.2
+- io.confluent:common-utils:5.5.2
+- org.glassfish.jersey.core:jersey-common:2.30
+
+The binary distribution of this product bundles these dependencies under the Eclipse Public License - v 2.0 (https://www.eclipse.org/org/documents/epl-2.0/EPL-2.0.txt)
```

Comment:
Sorry that I didn't catch this earlier.

Are we shipping a copy of the license to our users already? Usually, dependencies with other licenses are shipped like this: https://github.com/apache/flink/tree/master/flink-connectors/flink-connector-kinesis/src/main/resources/META-INF/licenses